libs/vkd3d/resource.c

```diff
@@ -2212,6 +2224,13 @@ static void d3d12_desc_update_bindless_descriptor(struct d3d12_desc *dst)
     is_buffer = dst->magic == VKD3D_DESCRIPTOR_MAGIC_CBV ||
             dst->u.view->type == VKD3D_VIEW_TYPE_BUFFER;
 
+    if ((dst->magic == VKD3D_DESCRIPTOR_MAGIC_SRV || dst->magic == VKD3D_DESCRIPTOR_MAGIC_UAV) &&
+        dst->u.view->weak && !dst->u.view->weak->alive)
+    {
+        WARN("Attempting to write descriptor %p which references a freed resource.\n", (const void*)dst);
```

Comment:
How common is this? Might be a lot of log spam since this is perfectly legal in D3D12, maybe ignore?